scripts/create-custom-scc.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# (c) Copyright [2021] Micro Focus or one of its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A script that will create a custom scc to run VerticaDB on OpenShift
+
+cat <<EOF | kubectl apply --validate=false -f -
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: anyuid-extra
+  annotations:
+    kubernetes.io/description: anyuid-extra provides all features of the anyuid SCC
+        but add SYS_CHROOT and AUDIT_WRITE capabilities.
+priority: 1
+requiredDropCapabilities:
+- MKNOD
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: MustRunAs
+fsGroup:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+allowedCapabilities:
+- SYS_CHROOT
+- AUDIT_WRITE
```

Comment:
If we don't have AUDIT_WRITE capabilities, will things still work?  We'll probably get messages like the audit facility is not working or something, but does it break the operator?  It would be nice to confirm if we just had SYS_CHROOT as the additional capability.

Reply:
I think it won't break the operator but I will check that again.

Reply:
It does break things when "AUDIT_WRITE" is removed. CreateDB fails.

Reply:
Okay, thanks for verifying.